cli/cmd/generate_oci.go

```diff
@@ -0,0 +1,446 @@
+package cmd
+
+import (
+	"time"
+
+	"github.com/imdario/mergo"
+	"github.com/spf13/cobra"
+
+	"github.com/AlecAivazis/survey/v2"
+	"github.com/lacework/go-sdk/lwgenerate/oci"
+	"github.com/pkg/errors"
+)
+
+type OciGenerateCommandExtraState struct {
+	AskAdvanced    bool
+	Output         string
+	TerraformApply bool
+}
+
+var (
+	// questions
+	QuestionOciEnableConfig            = "Enable configuration integration?"
+	QuestionOciTenantOcid              = "Specify the OCID of the tenant to be integrated"
+	QuestionOciUserEmail               = "Specify the email for the OCI user created for integration"
+	QuestionOciConfigAdvanced          = "Configure advanced integration options?"
+	QuestionCustomizeOciConfigName     = "Customize Config integration name?"
```

Comment:
nit: using config here and configuration for `QuestionOciEnableConfig`

Reply:
Granted I am likely reviewing what I had done previously in another provider 😆

Reply:
I'm not following. What's the ask here?

Reply:
So, I see in some of the strings we make use of the word config, and in others we use configuration. I think we should standardize around the use of configuration. Although perhaps folks from @lacework/growth-team have an opinion on this? 🤔

Reply:
Makes sense. I'll standardize around 'configuration' for now, but welcome input from others.